force-app/main/default/classes/NoteController.cls

```diff
@@ -53,29 +62,32 @@ public with sharing class NoteController {
     }
 
     @AuraEnabled
-    public static Note__c updateNote(String noteId, String changeLabelValue, String changeDescriptionValue){
+    public static Note__c updateNote(Id noteId, String changeLabelValue, String changeDescriptionValue){
         
         System.debug('Method updateNote: ============');
-        System.debug('nnteId: ' + noteId);
+        System.debug('noteId: ' + noteId);
         System.debug('changeLabelValue: ' + changeLabelValue);
         System.debug('changeDescriptionValue:' + changeDescriptionValue);
         
         
         Note__c note = [SELECT Id FROM Note__c WHERE Id = :noteId LIMIT 1];
-
-        if(note !=  null){
-            note.Label__c = changeLabelValue;
-            note.Description__c = changeDescriptionValue;
-            
-            update note; 
-            System.debug('Update========' + note); 
-        
-            return note;
-        }
-
-        System.debug('Note is not defined====');
-        return null;
-
+            try{
+                if(note !=  null){
+                    note.Label__c = changeLabelValue;
+                    note.Description__c = changeDescriptionValue;
+                    note.Created_Date__c = Datetime.now();
+                    
+                    update note; 
+                    System.debug('Update========' + note); 
+                
+                    return note;
+                }else{
+                    throw new AuraHandledException('Note not ');
```

Comment:
this error will be catched in row 87. For such exceptions please use custom exception. For example you need firstly just create custom Exception class (you can do it inside your current):
public class MyCustomException extends Exception {}

And later. use it in your code:
throw MyCustomException('erorr message');